packages/collector/src/agent/log.js

```diff
@@ -1,54 +0,0 @@
-/*
- * (c) Copyright IBM Corp. 2021
- * (c) Copyright Instana Inc. and contributors 2016
- */
-
-'use strict';
-
-const { http } = require('@instana/core').uninstrumentedHttp;
-const agentOpts = require('./opts');
-
-/**
- * @param {'debug' | 'info' | 'warning' | 'error'} logLevel
- * @param {string} message
- * @param {*} stackTrace
- */
-module.exports = function log(logLevel, message, stackTrace) {
-  /** @type {{m: string, st?: string}} */
-  const payloadObject = {
-    m: message.trim()
-  };
-  if (stackTrace) {
-    payloadObject.st = stackTrace.trim();
-  }
-
-  const payload = Buffer.from(JSON.stringify(payloadObject), 'utf8');
-
-  const req = http.request(
```

Comment:
It makes 0 sense to have another request handler for the agent requests.
The fn belongs to agentConnection downstreamConnection obj.
This instance is already aware of the agent config host etc.